### PR TITLE
Add a link to complete examples in documentation

### PR DESCRIPTION
**Pull request by mitmul**
I've got some feedbacks that some users want to see the complete working example code for some features. In such case, the small example code in the docstrings are not enough.

### CODE REVIEW

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by toslunar** [comment]
See also the discussion of #5388 on how to refer a code block in `examples`.

**Comment by mitmul** [comment]
@toslunar OK, this PR should be fixed after #4741 is merged.

**Comment by toslunar** [comment]
The PRs we mentioned have been merged.

File: chainer/serializers/npz.py [hunk 1]
```diff
@@ -59,6 +59,14 @@ def save_npz(file, obj, compression=True):
 
     This is a short-cut function to save only one object into an NPZ file.
 
+    See this example for how to use this function:
+    `MNIST custom loop example
+    <https://github.com/chainer/chainer/blob/master/examples/mnist/train_mnist_custom_loop.py#L56-L59>`_
```
Comment (himkt) [inline chainer/serializers/npz.py]: https://github.com/chainer/chainer/blob/master/examples/mnist/train_mnist_custom_loop.py#L123-L126 ?

File: chainer/serializers/npz.py [hunk 2]
```diff
@@ -163,6 +171,14 @@ def load_npz(file, obj, path='', strict=True, ignore_names=None):
     This is a short-cut function to load from an `.npz` file that contains only
     one object.
 
+    See this example for how to use this function:
+    `MNIST custom loop example
+    <https://github.com/chainer/chainer/blob/master/examples/mnist/train_mnist_custom_loop.py#L56-L59>`_
```
Comment (himkt) [inline chainer/serializers/npz.py]: ditto.

**Pull request closed without merging**